jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/GradleProjectProperties.java

```diff
@@ -39,8 +39,13 @@
 /** Obtains information about a Gradle {@link Project} that uses Jib. */
 class GradleProjectProperties implements ProjectProperties {
 
-  public static final String TOOL_NAME = "jib-gradle-plugin";
+  /** Used to generate the User-Agent header and history metadata. */
+  static final String TOOL_NAME = "jib-gradle-plugin";
+
+  /** Used for logging during main class inference. */
```

Comment:
Oh by purpose I meant like javadoc that says what the constant actually refers to. I think it'll be fine here, but in general, javadoc should be within the scope of the class and not refer to the actual actions of the caller/user of that class. For example, this constant refers to the name of the configuration object (extension) for Jib Gradle tasks, and the `JAR_PLUGIN_NAME` below is a description of where to configure the jar. 

If we were to have these be specifically used in main class reference, they should probably be grouped under a separate class for defining main class inference logging parameters.